Base/res/themes/Basalt.ini

```diff
@@ -74,6 +74,7 @@ TrayText=white
 
 [Flags]
 IsDark=true
+IsTitleCenter=false
```

Comment:
FWIW I think this would be even nicer as `TitleAlignment` with `Left`, `Center` and `Right` as possible values. But I'm not gonna ask you to implement that here, this is OK for now :)